benefits/static/css/styles.css

```diff
@@ -383,13 +382,19 @@ footer.global-footer .footer-links a:active {
 /* Enrollment Success */
 
 .enrollment-success #login {
-  padding: 0.3rem 0.6rem 0.1rem 0.6rem;
-  vertical-align: middle;
+  padding: 0.1rem 0.5rem 0.3rem 0.6rem;
+  display: inline-block;
+  padding-top: 0;
```

Comment:
does this `padding-top: 0` undo the `padding: 0.1rem ...` from above?

Reply:
will fix this in the next pr